masterlist.yaml

```diff
@@ -808,47 +906,112 @@ plugins:
     url: ['https://www.nexusmods.com/skyrimspecialedition/mods/2013/']
     after:
       - 'Verdant - A Skyrim Grass Plugin SSE Version.esp'
-  - name: 'JKs Whiterun.esp'
+  - name: 'JK''s Whiterun.esp'
```

Comment:
If this is because the plugin's name has changed, I'd turn it into a regex entry that matches both, e.g. `name: 'JK''?s Whiterun\.esp'`

Reply:
No it's a mistake.